developer-docs-site/docs/concepts/basics-events.md

```diff
@@ -3,9 +3,19 @@ title: "Events"
 slug: "basics-events"
 ---
 
-Events are emitted during the execution of a transaction. Each Move module can define its own events and choose when to emit the events upon execution of the module. For example, during a [coin transfer][coin_transfer], both the sender and receiver's accounts will emit `SentEvent` and `ReceivedEvent`, respectively. This data is stored within the ledger and can be queried via the REST interface's [Get events by event handle][get_events].
+# Events
 
-Assuming that an account `0xc40f1c9b9fdc204cf77f68c9bb7029b0abbe8ad9e5561f7794964076a4fbdcfd` had sent coins to another account, the following query could be made to the REST interface: `https://fullnode.devnet.aptoslabs.com/v1/accounts/c40f1c9b9fdc204cf77f68c9bb7029b0abbe8ad9e5561f7794964076a4fbdcfd/events/0x1::coin::CoinStore<0x1::aptos_coin::AptosCoin>/withdraw_events`. The output would be all `WithdrawEvent`s stored on that account, it would look like 
+Events are emitted during the execution of a transaction. Each Move module can define its own events and choose when to emit the events upon execution of the module. 
+
+## Example
+For example, during a [coin transfer][coin_transfer], the sender's account will emit  `SentEvent` and the receiver's account will emit `ReceivedEvent`. This event data is stored within the ledger and can be queried via the REST API's [Get events by event handle][get_events] endpoint.
```

Comment:
this is no longer accurate, we have DepositEvent anad WithdrawEvent as highlighted in the system integrators guide. If we want to update this page, we should do so with that guide.

Reply:
I removed this file from this PR. Will deal with it in a separate PR. I want to land this.